.env.example

```
@@ -78,14 +78,31 @@ OPENAI_REALTIME_VOICE=marin
 
 # ---- LLM ------------------------------------------------------------------
 # Choose: openai | github | anthropic | azure-aifoundry | mock | local
-LLM_PROVIDER=anthropic
+# azure-aifoundry (gpt-5.4-mini) is the production default.
+LLM_PROVIDER=azure-aifoundry
+LLM_MODEL=gpt-5.4-mini
+# Azure AI Foundry endpoint or Cloudflare AI Gateway route, e.g.
+# https://<resource>.openai.azure.com or
+# https://gateway.ai.cloudflare.com/v1/<account>/<gateway>/azure-openai/<resource>
+LLM_BASE_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LLM_BASE_URL key should go before the LLM_MODEL key

(UnorderedKey)